src/bootstrap.spec.ts

```diff
@@ -0,0 +1,11 @@
+import assert from 'assert'
+import { getBootstrapAddresses } from './bootstrap'
+
+describe('getBootstrapAddresses', function () {
+  it('passed addresses resolved first', async function () {
+    assert(await getBootstrapAddresses('/ip4/34.65.75.45/tcp/9091/p2p/16Uiu2HAm2cjqsDMmprtN2QKaq3LJrq3YK7vtdbQQFsxGrhLRoYsy'))
+  })
+  it('falls back to dns', async function () {
+    assert(await getBootstrapAddresses())
```

Comment:
Not sure what we are testing here, just that the function doesn't throw an error?

Would it be worth observe/spy `dns.promises.resolveTxt` to ensure it was called?

Reply:
yeah, it's kind of a dumb test, I could definitely make it test better.

Reply:
Will leave as is for now, as we don't use jest in this project so I'd have to add a mocking library.